accounts/models/connections.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from Gp_Backend.s3_storages import MediaStorage
+
+class connection(models.Model):
+    account = models.OneToOneField("accounts.Account", on_delete=models.CASCADE, related_name='connection')
+    name = models.CharField(max_length=100, null= False)
```

Comment:
but in `name` field `null=False` so must put `blank=False`?

Reply:
yes as `num=True` is required to accept a `null` value and this would happen if we don't set a value for the `name` field in the body request, `blank=true ` is required to accept no value like "" but it's not `null`